app/controllers/v0/institutions_controller.rb

```diff
@@ -45,9 +45,11 @@ def normalized_query_params
       query.tap do
         query[:name].try(:strip!)
         query[:name].try(:downcase!)
-        query[:state].try(:upcase!)
+        %i(state country).each do |k|
+          query[k].try(:upcase!)
+        end
         %i(type_name student_veteran_group yellow_ribbon_scholarship principles_of_excellence
```

Comment:
type_name is handled specially "school/employer" mapped to "not equal/equal OJT". All other filter fields here are booleans.